docs/src/api.nomnoml

```diff
@@ -0,0 +1,31 @@
+#title: api
```

Comment:
What is this file?

Reply:
It is the file used to generate the SVG diagram. This makes it possible for anyone to regenerate the SVGs without any specific software or knowledge (photoshop etc.)

Pasting this code into https://nomnoml.com/ or compiling it results into the SVGs included in this PR.

Do you think I should add a README with those files to give instructions on how to build them?

I think the code itself is pretty self-explanatory though, but I can look into adding comments if you think that would help.